lib/bridge/MatrixHandler.js

```diff
@@ -18,7 +18,7 @@ const MSG_PMS_DISABLED_FEDERATION = "[Bridge] Sorry, PMs are disabled on " +
 "this bridge over federation.";
 
 const KICK_RETRY_DELAY_MS = 15000;
-const KICK_DELAY_JITTER = 500;
+const KICK_DELAY_JITTER = 30000;
```

Comment:
err, delay_ms 30s, jitter 15s instead?

Reply:
I thought we wanted 30+/- 15?

Reply:
I think this is is mathematically equivalent, and easier to read.

Reply:
Travis has explained why I am wrong..but also why this approach works better.

Reply:
delay 15, jitter 30 (if jitter is additive here, which it is) is fine.

Reply:
> Travis has explained why I am wrong..but also why this approach works better.

For curious onlookers: the delay doesn't matter too much beyond giving a static offset to when kicks start. Having a 30s jitter means the spread of requests is much greater, which is a Good Thing for this. It has to be additive otherwise with a 15s delay a quarter of the values are negative. A 15s jitter instead gives too much concentration, making the problem worse. Having a 30s delay just means everything takes longer, so we shouldn't do that.

or TLDR: 30s is bigger than 15s for jitter, and we should do that. Additive because negative time is hard.